src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -162,6 +165,7 @@ class Renderer implements IListRenderer<CompletionItem, ISuggestionTemplateData>
 		const data = <ISuggestionTemplateData>templateData;
 		const suggestion = (<CompletionItem>element).completion;
 
+		data.root.id = getAriaId(_index);
```

Comment:
no more underscore for _index? (now that it is used...)

Reply:
Updated, thanks.